src/status_im/contexts/onboarding/share_usage/view.cljs

```diff
@@ -17,35 +17,38 @@
     (rf/dispatch [:navigate-to-within-stack [next-screen :screen/onboarding.share-usage]]) ;; Onboarding
     (rf/dispatch [:navigate-back]))) ;; Login Screen
 
+(defn- learn-more
+  []
+  (rf/dispatch [:show-bottom-sheet
+                {:content learn-more-sheet/view
+                 :shell?  true}]))
+
 (defn view
   []
   (let [insets           (safe-area/get-insets)
         next-screen      (:next-screen (rf/sub [:get-screen-params :screen/onboarding.share-usage]))
         share-usage-data (rn/use-callback #(share-usage-data-fn true next-screen))
-        maybe-later      (rn/use-callback #(share-usage-data-fn false next-screen))
-        learn-more       (rn/use-callback #(rf/dispatch [:show-bottom-sheet
-                                                         {:content learn-more-sheet/view
```

Comment:
Moved this fn to outside, we don't need the use-callback since this function is constant